src/docs/project_goals.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Project Goals
+---
+
+# Strategic Goals of Eclipse Theia
+
+This section describes the overall goals of the Eclipse Theia project.
+
+## Eclipse Theia mission statement: A framework for building tools and IDEs based on web technologies
+
+## An Open, Flexible and Extensible Tool Platform
+
+The goal of the Eclipse Theia project is to provide a platform for efficiently developing tools and IDEs based on a modern web technology stack. The primary target group for Theia is developers implementing a custom tool for end users. As Theia-based products are typically customized, branded and labeled by its adopters, the actual end users may not even be aware of the Theia project they are using under the hood. Therefore, end users are only an indirect secondary target group of Theia. 
+
+## Desktop & Browser, Local & Remote (Cloud)
+
+Tools and IDEs built on Theia can run as desktop applications or in the browser. The backend of Theia can run locally (for the desktop case) or as a remote service in the cloud. Both variants currently have an equal priority.
+
+## Platform for IDEs and Tools
+Theia is targeted at building a variety of custom tools. Prominent examples of Theia-based products include IDE's, but Theia also explicitly targets tools that do not focus on textual input/editors or software development. Thus, Theia's scope also included diagram editors or form-based UI. The Theia platform hence covers the typical application scenarios of the Eclipse rich-client platform (Eclipse RCP). As a consequence, the platform aims at making as few assumptions as possible about what adopters might want to build with Theia. It explicitly allows deviation from the standard workbench layout and the removal of all default features.
+
+## Basic Workbench Frame
+
+Theia provides a workbench frame, i.e., a window management system permitting the display of views, editors, and menus and makes available tools and interactions allowing the user to modify of the window layout, trigger commands, and use key bindings and other concepts known from a desktop tool such as drag and drop.
+
+## Reusable Common Tool Features
```

Comment:
minor: we might want a better title for this section.

Reply:
Do you have a suggestion?

Reply:
I'm not sure, perhaps `Reusable and Extensible Components`? If I understand correctly you're trying to describe how extensions which contribute functionality to the framework can be re-used across different products and can be extended to suit an application's potentially unique requirements. I did not understand the use of "tool features" in this context.

Reply:
Hmm, I am not sure about this. I personally find the fact that the platform provides features that you can resue important enough for a dedicated point. It is one of the core motivations to use a platform. Need a file explorer in your tool? Get it by just reusing it! So I would not mix "extensibility and adaptability" (section below) into this section.